packages/ace-cli/src/index.js

```diff
@@ -7,6 +9,7 @@ const winston = require('winston');
 
 const logger = require('@daisy/ace-logger');
 const ace = require('@daisy/ace-core');
+const util = require('@daisy/ace-core/lib/core/get-all-rules');
```

Comment:
at some point I'd like to put rules in their own package, but this is fine for now until it gets further modularized!